src/build.d

```diff
@@ -136,11 +139,14 @@ Command-line parameters
 
     auto targets = args
         .predefinedTargets // preprocess
+        .map!(t => cast(Dependency) t.ptr)
```

Comment:
Just have predefinedTargets return an array of Dependencies instead

Reply:
Done, just wanted to keep the initial diff as small as possible